### DISCUSSION TITLE
Guest Network

### DISCUSSION THREAD

**Discussion by TheCopperDragon** [travisghansen/hass-pfsense#92] [Q&A]
Hi Guys, 

I have a dedicated VLAN and SSID for my guest network. What I would like to do is monitor the guest VLAN via this integration to determine if there are any devices connected to it. I know I can track specified devices but would mean having to add guest devices manually. 
Is there a way to get a Boolean value to let me know there are devices connected or even an integer to give a count of the number of devices connected?

**Comment by travisghansen** [comment]
It’s not perfect but assuming you run dhcp on the guest network interface there is a count for the number of dhcp clients.

Another option is to possibly use the arp table data and put per-interface counts.

**Reply by TheCopperDragon** [reply]
@travisghansen 

Thank you for your answer.
I am using DHCP on all six of my VLANS so the DHCP client  count would be perfect for what I need if I am able to check a single VLAN.

I can see "sensor.pfsense_dhcp_leases_online" and "sensor.pfsense_dhcp_leases_total" entities within the addon but cannot see one for my guest VLAN specifically. Are you able to point me in the right direction please?

**Reply by travisghansen** [reply]
I was thinking those were broken down by nic but I guess not :( something that could be added though.
